health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/SearchChecklist.js

```
@@ -151,10 +150,7 @@ const SearchChecklist = () => {
               className={"boundaries-pop-module"}
               type={"default"}
               heading={t("CREATE_CHECKLIST")}
-              children={
-                [
-                ]
-              }
+              children={[]}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)